dev/.env.dev.example

```diff
@@ -41,3 +41,5 @@ RABBITMQ_PORT=5672
 RABBITMQ_DEFAULT_VHOST="/"
 
 REDIS_URI=redis://redis:6379/0
+
+DRF_SPECTACULAR_ENABLED=False
```

Comment:
I can see this can be handy when working on the frontend (to have swagger with newest contract locally exposed) so if it doesn't matter for you I can set to to True by default, wdyt?